nixos/modules/services/computing/slurm/slurm.nix

```diff
@@ -46,7 +61,17 @@ in
 
       client = {
         enable = mkEnableOption "slurm client daemon";
+      };
 
+      enableStools = mkOption {
+        type = types.bool;
+        default = false;
+        description = ''
+          Wether to provide a slurm.conf file.
+          Enable this option if you do not run a slurm daemon on this host
+          (i.e. <literal>server.enable</literal> and <literal>client.enable</literal> are <literal>false</literal>)
```

Comment:
Can we set the default for this option based on whether those other options are set?

Reply:
You mean if server || client enabled then set `enableStools` default to false? That can be done, but it would have no effect.

Reply:
The comment says to set it to true if the others are false, seems like that could be done in the default directly.

Reply:
The slurm module does not have a global `services.slurm.enable` option. Setting `enableStools` to default true would enable slurm for every nixos user out there.

Reply:
Ah, I see what you mean now.